### ISSUE TITLE
Links must have discernible text (.dropzone-showbox)

### ISSUE THREAD

**Issue by azaslonov** [paperbits/paperbits-demo#89]
Issue
Links must have discernible text (link-name)
Target application
API Management
Element path
.dropzone-showbox
Snippet
<a href="#" class="dropzone-showbox" alignment="center center" data-bind="alignment: position"> <div class="fit background-preview"></div> </a>
How to fix
Fix all of the following:
- Element is in tab order and does not have accessible text

Fix any of the following:
- Element does not have text that is visible to screen readers
- aria-label attribute does not exist or is empty
- aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
- Element's default semantics were not overridden with role="presentation"

**Comment by azaslonov** [comment]
Fixed.
